public/navigation.json

```diff
@@ -1739,6 +1739,13 @@
                   "type": "markdown",
                   "children": []
                 },
+                {
+                  "name": "How to keep attachments from original orders in subscriptions",
+                  "slug": "how-to-keep-attachments-from-original-orders-in-subscriptions#configuring-to-split-the-same-skus-with-different-attachments",
```

Comment:
É com o `#configuring-to-split-the-same-skus-with-different-attachments` também mesmo?

Reply:
Era não, corrigi e não tinha commitado, agora está certo, obrigada Karen!